src/api/ddoc/views.rst

```diff
@@ -315,6 +315,1144 @@ including the update sequence of the database from which the view was
 generated. The returned value can be compared this to the current update
 sequence exposed in the database information (returned by :get:`/{db}`).
 
+Search
+======
+
+Search indexes enable you to query a database by using `Lucene Query Parser Syntax <http://lucene.apache.org/core/4_3_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#Overview>`_. A search index uses one, or multiple, fields from your documents. You can use a search index to run queries, find documents based on the content they contain, or work with groups, facets, or geographical searches.
+
```

Comment:
I want to see a callout box here saying that Search is non-functional without a functioning, cluster-connected Clouesau instance, so that people don't search our docs for "Search" or "Lucene" and think the functionality is provided out of the box.

Sample callout box syntax is here, below the table (line 52): https://github.com/apache/couchdb-documentation/blob/299cc785bf1e631401d13444741d2fac80788513/src/setup/cluster.rst#ports-and-firewalls

Reply:
This still needs resolution.